google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerServiceContributor.java

```diff
@@ -46,6 +47,10 @@ public void contribute(StandardServiceRegistryBuilder serviceRegistryBuilder) {
         .applySetting("hibernate.connection.userAgent", HIBERNATE_API_CLIENT_LIB_TOKEN)
         // The custom Hibernate schema management tool for Spanner.
         .applySetting("hibernate.schema_management_tool", SCHEMA_MANAGEMENT_TOOL)
+        // Create a unique index for a table if it does not already exist when in UPDATE mode.
+        .applySetting(
+            "hibernate.schema_update.unique_constraint_strategy",
+            UniqueConstraintSchemaUpdateStrategy.RECREATE_QUIETLY)
```

Comment:
Why do we need to stray from the default here?

Reply:
The default strategy is to drop a unique index and create it in Spanner; it will produce the drop statement regardless of whether it exists or not for the table. However it the unique index does not exist then that will produce an error for Spanner.

The `recreate_quietly` strategy omits the initial drop operation at the beginning.